src/main/java/org/synyx/urlaubsverwaltung/absence/AbsencePeriod.java

```diff
@@ -0,0 +1,272 @@
+package org.synyx.urlaubsverwaltung.absence;
+
+import org.synyx.urlaubsverwaltung.person.Person;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Defines absence periods of a {@link Person}.
+ *
+ * e.g. Bruce Wayne is absent on:
+ * <ul>
+ *     <li>24.March 2021 to 28. March 2021 (vacation full day)</li>
+ *     <li>31.March 2021 (vacation morning)</li>
+ *     <li>9.June 2021 (vacation noon)</li>
+ *     <li>26.August 2021 to 27.August 2021 (sick full day)</li>
+ * </ul>
+ */
+public class AbsencePeriod {
+
+    public enum AbsenceType {
+        VACATION,
+        SICK,
+    }
+
+    public enum AbsenceStatus {
+        // vacation
+        WAITING,
+        TEMPORARY_ALLOWED,
+        ALLOWED,
+        ALLOWED_CANCELLATION_REQUESTED,
+        // sick note
+        ACTIVE,
+    }
+
+    private final Person person;
+    private final List<AbsencePeriod.Record> absenceRecords;
+
+    public AbsencePeriod(Person person, List<Record> absenceRecords) {
+        this.person = person;
+        this.absenceRecords = absenceRecords;
+    }
+
+    public Person getPerson() {
+        return person;
+    }
+
+    public List<AbsencePeriod.Record> getAbsenceRecords() {
+        return Collections.unmodifiableList(absenceRecords);
```

Comment:
List.of? https://stackoverflow.com/questions/39385658/list-of-or-collections-unmodifiablelist